automol/mult/ts.py

```diff
@@ -8,56 +7,59 @@
 from ._mult import spin as _spin
 
 
-def high(rct_mults, prd_mults):
-    """ Calculate high-spin multiplicity for a transition state
-        from the multiplicities of the reactants and products.
+def high(rct_mults: tuple[float], prd_mults: tuple[float]) -> int:
```

Comment:
Let's use `Sequence` instead of `tuple` for the input. Technically, spin multiplicities are always integers, but let's keep `float` for now.